### PR TITLE
Removed inappropriate effects for EFX on default sounds.

### PR DESCRIPTION
**Pull request by johncurley**
This is a temporary patch that lowers the default gain for EFX in line with EAX and also reverts AL_EFFECTSLOT_AUXILIARY_SEND_AUTO to false so it does not play on sound channels that it is not triggered for. This is a temporary patch until an actual working 3D sound implementation can be engineered as the original games did not have zone files and therefore no real way to zone audio effects.

### CODE REVIEW

**Comment by johncurley** [comment]
This patch should close #1278 until further improvements can be designed and implemented. For now it will cancel the reverb but it is not a true 3D sound environment due to the inability to trigger effects in the environment maps.

File: src/xrSound/SoundRender_Environment.cpp [hunk 1]
```diff
@@ -169,6 +175,7 @@ bool CSoundRender_Environment::load(IReader* fs)
 
         Room = fs->r_float();
         RoomHF = fs->r_float();
+        RoomLF = fs->r_float();
```
Comment (Xottab-DUTY) [inline src/xrSound/SoundRender_Environment.cpp]: This is binary file loading, it should not change!

Comment (Xottab-DUTY) [reply src/xrSound/SoundRender_Environment.cpp]: File is versioned, so order of loading values should not change.

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]
Please, revert changes to `CSoundRender_Environment::load`.

**Review by Xottab-DUTY — APPROVED** [review]

**Pull request merged**